### DISCUSSION TITLE
Commander PRO Plugin

### DISCUSSION THREAD

**Discussion by Fr0stX76** [Rem0o/FanControl.Releases#1226] [General]
@Rem0o 

You might want to remove the Commander PRO Plugin link from the readme.

Seems like the author deleted the page and closed his account...

https://github.com/iJacks1980/FanControl.CommanderPRO

https://github.com/ghost

**Comment by Rem0o** [comment]
Huh, odd, and unfortunate. That plugin had some traction, some guy even posted a complete youtube video on it: 
https://www.youtube.com/watch?v=TyxqEoWaBMU&t=1094s&ab_channel=CyberCPUTech

I believe he deleted the code from the repo at some point, which I found weird but I believe he said he would clean up the code or whatever and commit it back, but he had personal stuff going on. 

From an email notification on the deleted repo.
"I'm still working on it, sadly I've been dealing with some serious family issues which have taken most of my time and I'm going to be away for a few more days. But I'll try and get an interim update ready tonight (might be quite late)."

Hope the guy is ok. 

Will delete it for now.   I believe there is a single fork, but it's wayyyy out of date with what's in the wild and people are using.

**Reply by vdub12** [reply]
Thanks for the plug. That's my video. 

Do you know if anyone forked the plugin? Would be nice if anyone has the latest source to at least repost it. I have the binary but it's from back when I did the video. Not sure if it's the latest copy.

**Reply by Rem0o** [reply]
Haven’t found a recent fork unfortunately.

**Reply by Rem0o** [reply]
https://github.com/mmalewski/FanControl.CommanderPRO

**Reply by irlrobins** [reply]
Yep, saw his post about family issues, and then some insensitive idiot followed up a couple of days later with "remember when you have time to upload the update." So wouldn't surprise me if Steve decided to close up shop. Steve, if you're reading this, appreciate all the work you put in on this. Hope all ok in the real world.

**Comment by McT4V15h** [comment]
That youtube video was the main reason I purchased a commander pro, so I could do away with ICUE and the issues I have been having with their app just straight up crashing at the worlds worst times. Without the plugin for Fan control I have to ask, did I just waste my money or am I still going to be able to make things work, fan speeds etc with the commander pro? My guess is likely not which than leaves me pondering what to do next. Honestly I wish I would have found it sooner before it was deleted.. Now I am in a pickle..

**Reply by Rem0o** [reply]
I guess if somebody here has the binaries from a recent enough version, they could post it here.

**Reply by vdub12** [reply]
I have the binary from 7/9. I would be happy to share it. 
[FanControl.CommanderPro.zip](https://github.com/Rem0o/FanControl.Releases/files/9946327/FanControl.CommanderPro.zip)

**Reply by McT4V15h** [reply]
@vdub12 Thank you very much.. Your video is also an excellent help... I enjoy your content good sir. Please keep up the good work.

**Comment by EvanMulawski** [comment]
A replacement for that plugin is now available: https://github.com/EvanMulawski/FanControl.CorsairLink

**Reply by RoyRiv3r** [reply]
Do you have compiled version of this?

**Reply by EvanMulawski** [reply]
@Aghneel Yes, under Releases. https://github.com/EvanMulawski/FanControl.CorsairLink/releases

**Reply by vdub12** [reply]
Nice to see there's a replacement for the plugin. I'll check it out and maybe do another video on it. 

Thanks for taking this project on.
